Spring-Boot/Spring-Boot-Problem/src/test/java/tw/waterball/judgegirl/springboot/problem/controllers/ProblemControllerTest.java

```diff
@@ -268,5 +264,85 @@ void WhenSaveProblemWithTitle_ProblemShouldBeSavedAndRespondItsId() throws Excep
 
         assertEquals(randomTitle, requireNonNull(mongoTemplate.findById(id, Problem.class)).getTitle());
     }
+
+    @Test
+    void GivenOneProblemSavedAndPatchProblemWithNewTitle_WhenQueryTheSameProblem_ShouldHaveNewTitle() throws Exception {
+        Problem savedProblem = givenProblemsSaved(1).get(0);
```

Comment:
extract a method `givenProblemSaved()` instead of repeating `givenProblemsSaved(1).get(0);`